apps/extension/chromium/background-script/src/lib/service-worker/native-messaging-worker.ts

```
@@ -25,62 +42,137 @@ async function connect() {
-					console.error('Error responding to message', error);
-					sender.postMessage({ success: false, error: String(error) });
-				});
+			console.warn('Unknown action:', frame.action);
+			const errorFrame: Frame = {
+				kind: 'response',
+				id: frame.id,
+				action: frame.action,
+				event: '',
+				payload: undefined,
+				ok: false,
+			};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)